src/components/SidebarIndexTableOfContents.js

```diff
@@ -95,75 +98,50 @@ export function SidebarIndexTableOfContents({
     return null;
   }
 
-  /** Render the tree structure recursively */
+  /** */
   const renderTree = (node) => (
     <ScrollToForTreeItem
       containerRef={containerRef}
       key={node.id}
-      nodeId={node.id}
-      offsetTop={96} // offset for the height of the form above
+      itemId={node.id}
+      offsetTop={96}
       scrollTo={nodeIdToScrollTo === node.id}
     >
       <TreeItem
-        nodeId={node.id}
-        sx={{
```

Comment:
I need to visually review these deleted styles. The codemod might have done this bc I didn't intentionally delete all this.

Reply:
I visually reviewed this and found it to look identical to Mirador 3:


<img width="1035" alt="Screenshot 2025-02-12 at 5 32 09 PM" src="https://github.com/user-attachments/assets/bbc19251-8fac-4315-bd2c-f75588abd8b7" />